libs/server/Resp/RespServerSessionSlotVerify.cs

```diff
@@ -65,9 +55,10 @@ bool CanServeSlot(RespCommand cmd)
                             csvi.step = findRange.KeyStep;
                             break;
                         case FindKeysKeyNum:
+                            var beginSearch = (BeginSearchIndex)specs[0].BeginSearch;
                             var findKeysKeyNum = (FindKeysKeyNum)specs[0].FindKeys;
                             csvi.firstKey = searchIndex.Index + findKeysKeyNum.FirstKey - 1;
-                            csvi.lastKey = searchIndex.Index + parseState.GetInt(0);
+                            csvi.lastKey = searchIndex.Index + parseState.GetInt(beginSearch.Index + findKeysKeyNum.KeyNumIdx - 1);
```

Comment:
this is wrong, need to merge from the recently merged PR.

Reply:
This line should be: `csvi.lastKey = csvi.firstKey + parseState.GetInt(searchIndex.Index + findKeysKeyNum.KeyNumIdx - 1);`

Reply:
Fixed the issue of wrong merge